rules/linux/auditd/lnx_auditd_clean_disable_dmesg_logs_via_syslog.yml

```
@@ -0,0 +1,40 @@
+title: Clear Or Disable Dmesg Logs via Syslog Syscall - Linux
+id: eca5e022-d368-4043-98e5-9736fb01f72f
+status: experimental
+description: |
+    Detects the use of the `syslog` syscall with action code 5 (SYSLOG_ACTION_CLEAR),
+    (4 is SYSLOG_ACTION_READ_CLEAR and 6 is SYSLOG_ACTION_CONSOLE_OFF) which clears the kernel
+    ring buffer (dmesg logs). This can be used by attackers to hide traces after exploitation
+    or privilege escalation. A common technique is running `dmesg -c`, which triggers this syscall internally.
+references:
+    - https://man7.org/linux/man-pages/man2/syslog.2.html
+    - https://man7.org/linux/man-pages/man1/dmesg.1.html
+author: Milad Cheraghi
+date: 2025-05-27
+tags:
+    - attack.defense-evasion
+    - attack.t1070.003
```

```suggestion
    - attack.t1070.002
```
I think it is more relevant